packages/mobile/src/geth/saga.ts

```diff
@@ -114,7 +114,7 @@ export function* waitForNextBlock() {
 function* waitForGethInstance() {
   try {
     const fornoMode = yield select(fornoSelector)
-    // get geth without syncing if fornoMode
+    // get geth without starting if fornoMode
```

Comment:
With the change, we will no longer need to `waitForGethInstance` in forno mode

Reply:
Resolving based on our chat